modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -83,21 +113,52 @@ spec:
           name: cloud-credentials
         default: true
         objectStorage:
-          bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          bucket: my-bucket
+          prefix: velero
         provider: aws
   configuration:
     restic:
       enable: <true_or_false>
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14" <4>
+      volumeOptions: <5>
+        sourceVolumeOptions:
+          accessMode: ReadOnlyMany
+          cacheAccessMode: ReadWriteOnce
+          cacheCapacity: 2Gi
+          moverSecurityContext: true
+        destinationVolumeOptions:
+          storageClass: other-storageclass-name
+          cacheAccessMode: ReadWriteMany
+          moverSecurityContext: true
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-west-2
+        provider: aws
+
 ----
-<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
+<1> OADP 1.2.x only.
+<2> OADP 1.2.x only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for backup. The default value is 10.
+<3> OADP 1.2.x only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for restore. The default value is 10.
+<4> OADP 1.2.x only. Optional: Specify the number of days, between running restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption, from no longer referenced data, and access costs.
+<5> OADP 1.2.x only. Optional: Specify VolumeSync volume options for backup and restore.
+
 +
-The OADP Operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
+. The OADP Operator installs two Custom Resource Definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
 +
```

Comment:
I would stick with the original here -- "The OADP Operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`."

You added a period at the start of the line, btw. It should not be there.